ui/apps/dashboard/src/app/(dashboard)/env/[environmentSlug]/apps/[externalID]/AppCard/AppCard.tsx

```diff
@@ -0,0 +1,106 @@
+'use client';
+
+import { classNames } from '@inngest/components/utils/classNames';
+
+import { FrameworkInfo } from '@/components/FrameworkInfo';
+import { LanguageInfo } from '@/components/LanguageInfo';
+import { SyncStatus } from '@/components/SyncStatus';
+import { Time } from '@/components/Time';
+import { PlatformSection } from './PlatformSection';
+
+type Props = {
+  app: App;
+  className?: string;
+};
+
+type App = {
+  externalID: string;
+  latestSync: Sync | null;
+  name: string;
+};
+
+type Sync = {
+  createdAt: Date;
+  framework: string | null;
+  sdkLanguage: string | null;
+  sdkVersion: string | null;
+  status: string;
+  url: string | null;
+} & React.ComponentProps<typeof PlatformSection>['sync'];
+
+export function AppCard({ app, className }: Props) {
```

Comment:
**suggestion**: should this be called something else? Maybe `AppInfoCard`? I got confused a few times by the already existing `AppCard` component.